src/mlpack/methods/ann/ffn.hpp

```diff
@@ -25,6 +25,7 @@
 #include <mlpack/methods/ann/layer/layer_types.hpp>
 #include <mlpack/methods/ann/init_rules/random_init.hpp>
 #include <mlpack/methods/ann/init_rules/nguyen_widrow_init.hpp>
+#include <mlpack/methods/ann/init_rules/kathirvalavakumar_subavathi_init.hpp>
```

Comment:
I think as long as we don't use the kathirvalavakumar subavathi initialization method in the FFN class, we don' have to include the header file, I guess the same applies for the nguyen widrow method.